app/src/main/res/font/metropolis.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8"?>
+    <font-family xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto">
+
+        <!-- Black -->
+        <font
+            android:font="@font/metropolis_black"
+            android:fontStyle="normal"
+            android:fontWeight="900"
+            app:fontStyle="normal"
+            app:fontWeight="900"
+            app:font="@font/metropolis_black"
```

Comment:
Would you mind adding a comment at the top of this file explaining why we need to duplicate these settings?